WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -1530,6 +1530,19 @@ local methods = {
     -- no addon, or no data, or ignore flag
     return false, false, nil, nil
   end,
+  ["RefreshBT2UpgradeIcon"] = function(self)
+    if not self.data.controlledChildren and self.data.triggers then
+      for i, t in pairs(self.data.triggers) do
+          if type(i) == "number" then
```

Comment:
You don't really have to check that, as you don't use i.

Reply:
It is necessary if using pairs instead of ipairs :)

But you already mentioned that.